projects/plutus/test/test_current_sense.c

```diff
@@ -0,0 +1,86 @@
+#include <inttypes.h>
+
+#include "current_sense.h"
+
+#include "critical_section.h"
+#include "gpio.h"
+#include "interrupt.h"
+#include "log.h"
+#include "ltc2484.h"
+#include "soft_timer.h"
+#include "test_helpers.h"
+#include "unity.h"
+
+// Arbitrary number of testing samples
+#define TEST_NUM_SAMPLES 25
+
+// Mock SPI data
+static uint32_t x = 0;
+StatusCode TEST_MOCK(spi_exchange)(SPIPort spi, uint8_t *tx_data, size_t tx_len, uint8_t *rx_data,
+                                   size_t rx_len) {
+  if (rx_data != NULL) {
```

Comment:
Why are you mocking SPI data? You don't know the format of the ADC's SPI communications.

Reply:
Removed